Makefile.old

```diff
@@ -0,0 +1,2 @@
+test:
```

Comment:
Could we this to the same Makefile but under a different target? Something like `python-test` for instance.